README.md

```diff
@@ -130,20 +130,36 @@ page.driver.header 'Referer', 'https://www.thoughtbot.com'
 Contributing
 ------------
 
-See the CONTRIBUTING document.
+See the [CONTRIBUTING] document.
+Thank you, [contributors]!
+
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/capybara-webkit/graphs/contributors
+
+License
+-------
+
+capybara-webkit is Copyright (c) 2010-2015 thoughtbot, inc. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
 
 About
 -----
 
-The capybara WebKit driver is maintained by Joe Ferris and Matt Horan. It was written by [thoughtbot, inc](http://thoughtbot.com/community) with the help of numerous [contributions from the open source community](https://github.com/thoughtbot/capybara-webkit/contributors).
+The capybara WebKit driver is maintained by Joe Ferris and Matt Horan.
 
-Code for rendering the current webpage to a PNG is borrowed from Phantom.js' implementation.
+Code for rendering the current webpage to a PNG is borrowed from Phantom.js'
+implementation.
 
-![thoughtbot](http://thoughtbot.com/images/tm/logo.png)
+![thoughtbot](https://thoughtbot.com/logo.png)
```

Comment:
This logo is out of date. New version is http://thoughtbot.github.io/images/signature.svg